src/Kubernetes.WebSocket.cs

```diff
@@ -67,16 +72,24 @@ public Task<WebSocket> WebSocketNamespacedPodExecAsync(string name, string @name
 
             uriBuilder.Path += $"api/v1/namespaces/{@namespace}/pods/{name}/exec";
 
+            List<string> queryParameters = new List<string>();
 
-            uriBuilder.Query = QueryHelpers.AddQueryString(string.Empty, new Dictionary<string, string>
+            foreach (var c in command)
             {
-                { "command", command},
-                { "container", container},
-                { "stderr", stderr ? "1": "0"},
-                { "stdin", stdin ? "1": "0"},
-                { "stdout", stdout ? "1": "0"},
-                { "tty", tty ? "1": "0"}
-            }).TrimStart('?');
```

Comment:
that `trimstart ?` is because on net452, this api will NOT return with a `?` prefix (different from netstanard)

Reply:
Yeah, let's maintain this. (and is this covered by a test-case? if it's not it should be...)